lib/content-services/src/lib/tag/tags-creator/tags-creator.component.scss

```diff
@@ -27,13 +37,37 @@ adf-tags-creator {
             padding-right: 3px;
             font-size: 14px;
         }
+
+        .mat-form-field-appearance-fill .mat-form-field-flex {
```

Comment:
Same here internal material classes used which will be an issue for Angular upgrade

Reply:
@MichalKinas The Denys has asked for the same. Here is the conversation you can check.
https://github.com/Alfresco/alfresco-ng2-components/pull/8837#discussion_r1335988859